### DISCUSSION TITLE
Uganda Community's Mapping Competition /w StreetComplete

### DISCUSSION THREAD

**Discussion by RubenKelevra** [streetcomplete/StreetComplete#4861] [General]
The Open Street Map community in Uganda is organizing a mapping competition to celebrate the Woman's Day, using the StreetComplete app.

I thought I'll share this here with the community:

![Screenshot_2023-03-06-16-22-23-288-edit_com.twitter.android.jpg](https://user-images.githubusercontent.com/614929/223153525-7144e9bd-2257-4694-87f3-544c0059f552.jpg)

[Link](https://twitter.com/osmuganda/status/1632711769025028102)

**Comment by matkoniecz** [comment]
Hope that it will go well. I just hope that they will focus more on "mapping" than "competition" to avoid people adding guesses.

**Reply by RubenKelevra** [reply]
Nah, I wouldn't expect that to be the focus for this, sounds to me like a friendly competition, just to have fun together.

I've just explained them that using the team mode would be helpful, if there are groups mapping together to avoid unnecessary refreshes of the data leading to more mobile traffic beeing used.

:)

**Comment by RubenKelevra** [comment]
Here's the winner :)

![Screenshot_2023-03-20-13-44-27-564-edit_com twitter android](https://user-images.githubusercontent.com/614929/226342440-04b08faa-c608-4f7f-ba27-670440436a2a.jpg)


[Tweet](https://twitter.com/osmuganda/status/1637763944017276928)
